docs/contributing/site-showcase-submissions.md

```diff
@@ -50,7 +50,7 @@ Use the following template to ensure required fields are filled:
 - title: (required)
   url: (required)
   main_url: (required)
-  source_url: (optional - https://github.com/{username}/{titleofthesite})
+  source_url: (optional - https://github.com/username/titleofthesite)
```

Comment:
```suggestion
  source_url: (optional - https://github.com/{username}/{repository(_name)})
```
What do we think of something like this? Do you still have the same inclination about replacement?

I worry that if we drop all the curly braces it doesn't draw your attention to the changes you need to make in the template. Another option is <> but I'm not sure that would have solved the misunderstanding.

Reply:
I see your point. Let's maybe do it more KISS. I suggest revert back all changes besides line 31 => https://github.com/gatsbyjs/gatsby/pull/20373/commits/721147356b2e55265506a1975f49c206be9194b2#diff-94bc22b42ee6b592b2c39e07adc7cbe0R31. To be honest, the line 31 makes the biggest confusion. Otherwise, we can end up with something more complicated and not clear. @laurieontech wdty?

Reply:
That makes sense to me. Would you mind making the update and then we can merge it in? Thanks for the discussion on this!

Reply:
Done.